mock/world/gogoProtoMarshalizer.go

```diff
@@ -0,0 +1,51 @@
+package worldmock
```

Comment:
Another TODO to remove this when we will have the elrond-go-core

Reply:
done